.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: release
+on:
+  release:
+    types: [published]
+jobs:
+  process:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@v2
+        with:
+          ref: master
+      - name: modif docs
+        run: |
+          release="${{ github.event.release.tag_name }}"
```

Comment:
Here I'm not sure whether this makes sense: the modification of the docs should happen before defining the tag and not after? So I think this cannot be automatised except maybe with an additional `make release` command or similar, that can be executed by the person taking the release...

Reply:
This step modifies the source doc file `index.rst` before dispatching the action to `gammapy-docs` to build the docs. The *release tagging* for the docs is done on `gammapy-docs` repo, which places the sphinx-built docs in its respective version release folder. We could certainly skip this step, to have the good source doc file `index.rst` living in release tag of the code repo, which implies having an additional non-critical manual step and believe that it is going to be properly done. 

I understand that the release tagging is done manually in Github, this triggers the Github action to automates the steps. Another approach could be having a `make release` command that do a POST request to Github API to tag release. In this case, we could make the modifs in `index.rst` as well as sending the two `curls -XPOST` of the dispatched actions with a script called by the `make release` command. I think in this approach we should have to consider dealing with permissions to tag/make a release via the Github API.

Reply:
Is there maybe a solution, so that the docs do not mention a specific release at all? I just checked and no other package that is like Gammapy mentions a specific release in the install instructions (I checked e.g. Numpy, Astropy or Scipy). Maybe we keep the normal installation instructions general like `conda install gammapy` or `pip install gammapy` and then we create a dedicated section "Conda Environments", which shows the process using a wildcard for the version?

Reply:
I'd be in favor of simplifying the install process, advertising mainly `conda install gammapy` or `pip install gammapy` We could keep the env yaml files for `gammapy download notebooks` to guarantee reproducibility, and as an alternative less visible in the docs option for install. Moreover, we could also provide Docker images in the Docker Hub.

This said I would not provide commands that will not work when _copy pasted_, so I would not use wildcards.

Reply:
@adonath 
I think we can keep this step, we talked about removing it and do it manually before tagging the release, but I think it does not hurt to keep it. So in case we forgot to do it manually, this part of the text in the release version of the docs will be fixed anyway. And in the case it is done manually, there will be no modifs, since it will replace the text with the same text.

Reply:
Yes, that's a fair point...